eZ/Publish/Core/Search/Common/FieldValueMapper/Aggregate.php

```diff
@@ -34,24 +41,16 @@ public function __construct(array $mappers = [])
         }
     }
 
-    /**
-     * Adds mapper.
-     *
-     * @param \eZ\Publish\Core\Search\Common\FieldValueMapper $mapper
-     */
-    public function addMapper(FieldValueMapper $mapper)
+    public function addMapper(FieldValueMapper $mapper, ?string $searchTypeFQCN = null): void
```

Comment:
This can be described (check that I used the right class)

```suggestion
    /**
     * @param class-string<\eZ\Publish\SPI\Search\Field>|null $searchTypeFQCN
     */
    public function addMapper(FieldValueMapper $mapper, ?string $searchTypeFQCN = null): void
```